blueprints/devops/goreleaser/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: goreleaser
+matrix:
+  architectures:
+    - amd64
+  versions:
+    - 2.0.1
+homepage: https://goreleaser.com
+summary: release Go projects as fast and easily as possible
+description: |-
+  GoReleaser is an open-source tool that simplifies the process of releasing your
+  Go projects to multiple platforms, ensuring a consistent and reliable
+  deployment experience.
+fetch:
+  url: https://github.com/goreleaser/goreleaser/releases/download/v{{version}}/goreleaser_Linux_{{target}}.tar.gz
```

Comment:
Hey @fyhertz what is the difference between `{{target}}` and `{{arch}}` ? Are they aliases ?

Reply:
`arch` is the debian `arch` (amd64, arm64, armhf) and `target` can be set to anything using the `targets` maps ! In this case, you can simply use `arch` since the releases already use debian architectures.